shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/BaseRule.g4

```diff
@@ -416,6 +416,7 @@ simpleExpr
     | ROW? LP_ expr (COMMA_ expr)* RP_
     | EXISTS? subquery
     | LBE_ identifier expr RBE_
+    | (owner DOT_)? name LBT_ expr (COMMA_ expr)* RBT_ simpleExpr?
```

Comment:
Where is the definition of modelRulesClause's expr

Reply:
I added `(owner DOT_)? name LBT_ expr (COMMA_ expr)* RBT_ simpleExpr?` rule to support for expressions coming from `modelRulesClause`'s right-hand side assignment.

![modelRulesClauseDefinition](https://user-images.githubusercontent.com/48581379/127779755-900bcd33-d2ce-44d7-9f74-feac2c1a3838.png)

![modelRulesClauseRule](https://user-images.githubusercontent.com/48581379/127779768-9f514565-4414-43f2-944d-f4bda990b61c.png)

I found out the actual [definition](https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/Model-Expressions.html#GUID-83D3FD56-8346-4D3F-A49E-5FE41FE19257) of `modelRulesClause`'s right-hand side assignment. 

![RightHandsideRule](https://user-images.githubusercontent.com/48581379/127779923-8fef8e16-9b46-41ba-a50f-cbc98cffbd67.png)

I'll remove the rule `(owner DOT_)? name LBT_ expr (COMMA_ expr)* RBT_ simpleExpr?` as it's not appropriate and update the `modelRuleClause`'s right-hand side `expr` to `modelExpr`.

Reply:
That's great.